ws/filetransfer/model.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package filetransfer
+
+type MenderFileTransferMessageStatus int
+
+const (
+	MessageTypeGetFile   = "get_file"
+	MessageTypePutFile   = "put_file"
+	MessageTypeStatFile  = "stat_file"
+	MessageTypeFileChunk = "file_chunk"
+)
+
+const (
+	NormalMessage MenderFileTransferMessageStatus = iota + 1
+	ErrorMessage
+)
```

Comment:
```suggestion
type MessageType string

const (
	MessageTypeGetFile   MessageType = "get_file"
	MessageTypePutFile   MessageType = "put_file"
	MessageTypeStatFile  MessageType = "stat_file"
	MessageTypeFileChunk MessageType = "file_chunk"
	MessageTypeError     MessageType = "error"
)

type Error struct {
	Error       string `msgpack:"err",json:"error"`
	MessageType MessageType `msgpack:"msgtype,omitempty",json:"message_type"`
}
```

Reply:
This is more like what I had in mind...

Reply:
there is really no point, we are putting them into [MsgType](https://github.com/mendersoftware/go-lib-micro/blob/master/ws/protomsg.go#L37) which is string, besides lets for now be compatible with the ws/shell maybe?

Reply:
and Error and NormalMessage are passed in the Properties array in "startus" key also as strings.

Reply:
I was hoping we could avoid that - using the "status" key, I mean.

Reply:
for now I would keep it in the same way as we do with the shell, we have mostly everything in place and we know it works. you know no need to change something that works, right?

Reply:
I was hoping we could make this change now so we don't have to break the API in the future...

Reply:
lets discuss after su. it means changes to the UI and it will diverge from what we know and have for mender shell. also we will have shell that uses MsgProto in a different way than file transfer add-on.

Reply:
discussed separately.